packages/tests/package.json

```diff
@@ -35,9 +34,31 @@
       ],
       "output": [
         "utils",
+        "rollup-resolve-remap.{js,js.map}",
+        "run-web-tests.{js,js.map}",
+        "web-test-runner.config.{js,js.map}",
+        "wtr-config.{js,js.map}",
         "tsconfig.tsbuildinfo"
       ],
       "clean": "if-file-deleted"
+    },
+    "test": {
+      "dependencies": [
+        "test:dev",
+        "test:prod"
+      ]
+    },
+    "test:dev": {
+      "command": "node ./run-web-tests.js",
+      "dependencies": [
+        "build"
```

Comment:
I think `test:dev` and `test:prod` should depend on all of the packages that the monolithic test config tests. And then if we add `files:[], output:[]` we will be able to cache all browser tests as long as no browser packages changed.

Reply:
How about I do this in a follow-up? I think we should include the node tests here too, so given the way we might want to separate tests on runners, we might want `test`', `test:browser`, `test:node`, `test:browser:dev`, etc. Seem worth a little discussion.

Reply:
Follow up sounds good. Surprised that we would want to run node tests in here, though. I think of this package as "the monolithic test package that only exists because it's too slow/flakey to run browser tests in isolation".

Reply:
I guess we want to remove running all tests here and move that to the top-level package.json. We need the WTR --no-tunnel flag for that though, so I can do what you recommended here in the meantime. I'll still do a follow-up though.